pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -79,9 +83,16 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 	for _, sc := range aiConfig.StorageConfigList {
 		// Check that file is verified
 		locationDir := finalDirname + "/" + sc.ImageSha256
-		location, err := locationFromDir(locationDir)
-		if err != nil {
-			return err
+		location := ""
+		err := errors.New("")
```

Comment:
At last this should be an "var err errors", but I suspect you can get rid of it and line 90, and change = to := down below where err needs to be defined.

Reply:
This was done in my previous commit..Diff needs to be updated,